Cannot start React Server website

_This was introduced by a change in Asini version. It's not clear to me if it's an Asini bug or if we need to update the instructions to start the service._

### Reproduction
1. Follow the instructions in the [react-server-website README](https://github.com/redfin/react-server/blob/master/packages/react-server-website/README.md):
```
$ git clone git@github.com:redfin/react-server.git
$ cd react-server
$ npm run bootstrap
$ cd packages/react-server-website
$ npm run build-assets
```

### Bug
```
$ npm run build-assets

> react-server-website@0.5.1 build-assets /Users/david.alber/tmp/rstest/react-server/packages/react-server-we$
site
> npm run docs && NODE_ENV=production react-server compile


> react-server-website@0.5.1 docs /Users/david.alber/tmp/rstest/react-server/packages/react-server-website
> docco -o docs ./components/*.js ./lib/*.js ./middleware/*.js ./pages/*.js *.js && node build-dir-contents.js

sh: docco: command not found

npm ERR! Darwin 15.6.0
npm ERR! argv "/Users/david.alber/.nvm/versions/node/v6.9.4/bin/node" "/Users/david.alber/.nvm/versions/node/v6.9.4/bin/npm" "run" "docs"
npm ERR! node v6.9.4
npm ERR! npm  v3.10.10
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! react-server-website@0.5.1 docs: `docco -o docs ./components/*.js ./lib/*.js ./middleware/*.js ./pages/*.js *.js && node build-dir-contents.js`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the react-server-website@0.5.1 docs script 'docco -o docs ./components/*.js ./lib/*.js ./middleware/*.js ./pages/*.js *.js && node build-dir-contents.js'.
```

This was introduced in 7f11ef17d6acd7d2fc1fba960f28b2d7c1f70447.

--- a/package.json
+++ b/package.json
@@ -12,7 +12,7 @@
     "update-dependencies": "david u && asini exec david u"
   },
   "devDependencies": {
-    "asini": "1.4.0",
+    "asini": "1.3.0",
     "asini-changelog": "^1.1.0",
     "david": "^10.0.0"
   },

